traversal/planner/GraphPlanner.java

```diff
@@ -107,55 +100,6 @@ static GraphPlanner create(Structure structure) {
         return planner;
     }
 
-    @Override
-    public GraphProcedure procedure() {
-        assert procedure != null;
-        return procedure;
-    }
-
-    @Override
-    public boolean isGraph() {
-        return true;
-    }
-
-    @Override
-    public GraphPlanner asGraph() {
-        return this;
-    }
-
-    public Collection<PlannerVertex<?>> vertices() {
-        return vertices.values();
-    }
-
-    public Set<PlannerEdge<?, ?>> edges() {
-        return edges;
-    }
-
-    void setOutOfDate() {
-        this.isUpToDate = false;
-    }
-
-    private boolean isUpToDate() {
-        return isUpToDate;
-    }
-
-    private boolean isPlanned() {
-        return optimiser.isFeasible() || optimiser.isOptimal();
-    }
-
-    @Override
-    public boolean isOptimal() {
-        return optimiser.isOptimal();
-    }
-
-    private boolean isError() {
-        return optimiser.isError();
-    }
-
-    Optimiser optimiser() {
-        return optimiser;
-    }
-
```

Comment:
This looks like a bigger change than it is. All the creation routines are now pushed together and the accessors were moved further down in the file.